autopush/gcdump.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python
+# type: ignore
```

Comment:
(mypy doesn't support this yet, but will eventually cause entire file to be ignored)